README.md

```diff
@@ -59,8 +59,8 @@ environment for PyTorch:
 
     ```bash
     ml --force purge
-    ml Python CMake/3.24.3-GCCcore-11.3.0 mpi4py OpenMPI CUDA/11.7
-    ml GCCcore/11.3.0 NCCL/2.12.12-GCCcore-11.3.0-CUDA-11.7.0 cuDNN
+    ml Python CMake/3.24.3-GCCcore-11.3.0 mpi4py OpenMPI CUDA/12.3
```

Comment:
One thing I have been thinking about is whether we should enforce `Python/3.11.3` instead of just `Python` here. This is not directly related to your PR, but I would like your opinion on changing this across the code base. We could do this in a different branch or something, in which case. Doing it here would also be fine though.

Reply:
you're right

Reply:
So do you think I should change this here or should I create an issue to change this across the codebase?

Reply:
This will not necessarily remain at Python 3.11 in the next stage. We will soon have Stage 2025, when the version will potentially change. And the new stage will probably provide the latest Python version. For HPC centers, these scripts have to be anyway updated with time.

Reply:
ok, so for now let's leave it as is